### PR TITLE
Refactor resgroup cpu_rate_limit test.

### PR DESCRIPTION
**Pull request by kainwen**

### CODE REVIEW

File: src/test/isolation2/input/resgroup/resgroup_cpu_rate_limit.source [hunk 1]
```diff
@@ -1,52 +1,62 @@
...
+conn = pg.connect(dbname="isolation2resgrouptest")
+group_cpus = conn.query("select rsgname, cpu_usage from gp_toolkit.gp_resgroup_status")\
+                 .getresult()
+json_text = json.dumps(dict([(name, json.loads(cpu)) for name, cpu in group_cpus]))
+sql = "insert into cpu_usage_samples values ('{value}')".format(value=json_text)
+conn.query(sql)
+return json_text
+$$ LANGUAGE plpythonu;
+
+CREATE OR REPLACE FUNCTION verify_cpu_usage(groupname TEXT, expect_cpu_usage INT, err_rate INT)
```
Comment (asimrp) [inline src/test/isolation2/input/resgroup/resgroup_cpu_rate_limit.source]: This function needs a comment describing the intention and how it's accomplished.

Comment (kainwen) [reply src/test/isolation2/input/resgroup/resgroup_cpu_rate_limit.source]: OK. I will add some comment. Thanks.

File: src/test/isolation2/output/resgroup/resgroup_cpu_rate_limit.source [hunk 2]
```diff
@@ -51,11 +55,11 @@ CREATE
 -- shares := 1024 * gp_resource_group_cpu_priority
 --
 
-! python -c "print $(cat @cgroup_mnt_point@/cpu/gpdb/cpu.cfs_quota_us) == int($(cat @cgroup_mnt_point@/cpu/gpdb/cpu.cfs_period_us) * $(nproc) * $(psql -d isolation2resgrouptest -Aqtc "SHOW gp_resource_group_cpu_limit"))";
+! python -c "print $(cat /sys/fs/cgroup/cpu/gpdb/cpu.cfs_quota_us) == int($(cat /sys/fs/cgroup/cpu/gpdb/cpu.cfs_period_us) * $(nproc) * $(psql -d isolation2resgrouptest -Aqtc "SHOW gp_resource_group_cpu_limit"))";
```
Comment (pivotal-ning-yu) [inline src/test/isolation2/output/resgroup/resgroup_cpu_rate_limit.source]: Please keep the `@cgroup_mnt_point@` placeholder here.

Comment (pivotal-ning-yu) [reply src/test/isolation2/output/resgroup/resgroup_cpu_rate_limit.source]: and other places.

Comment (kainwen) [reply src/test/isolation2/output/resgroup/resgroup_cpu_rate_limit.source]: Thanks for pointing out the problem. I will fix it.

**Pull request merged**